### ISSUE TITLE
Error: not enough block information when running from terminal

### ISSUE THREAD

**Issue by kennyng** [SelfControlApp/selfcontrol#273]
After setting the BlockDuration, HostBlacklist, and BlockAsWhitelist default values, running the command to start the block (sudo /Applications/SelfControl.app/Contents/MacOS/org.eyebeam.SelfControl $(id -u $(whoami)) --install) does not work. 

I get the following error:
```
2015-03-12 13:19:23.564 org.eyebeam.SelfControl[2963:57721] Using your own bundle identifier as an NSUserDefaults suite name does not make sense and will not work. Break on _NSUserDefaults_Log_Nonsensical_Suites to find this
2015-03-12 13:19:23.582 org.eyebeam.SelfControl[2963:57721] ERROR: Not enough block information.
```

What block information am I missing? Are there other values I need to set before running SelfControl via Terminal?

EDIT:
Although the FAQ says not to, after setting the BlockStartedDate to the current datetime + 1 minute, the block was successfully added.
In HelperMain.m, the following code checks if BlockStartedDate == [NSDate distantFuture]. 
```
if([lockDictionary[@"HostBlacklist"] count] <= 0 || [lockDictionary[@"BlockDuration"] intValue] < 1
    || lockDictionary[@"BlockStartedDate"] == nil
    || [lockDictionary[@"BlockStartedDate"] isEqualToDate: [NSDate distantFuture]]) {
        NSLog(@"ERROR: Not enough block information.");
        printStatus(-210);
        exit(EX_CONFIG);
}
```

In AppController.m, the BlockStartedDate is set in line 599: `[defaults_ setObject: [NSDate date] forKey: @"BlockStartedDate"];`.
However, I am not sure if BlockStartedDate is also being set when SelfControl is started via command line.

**Comment by aly006** [comment]
I have the same issue reported as above when attempting to start selfcontrol through the command line.  There really should be a sane method to automatically schedule selfcontrol to run.

**Comment by cstigler** [comment]
should be fixed in #284, please try updating to v2.1 and reopen if there are still issues

**Comment by slambert** [comment]
@aly006 Check out this? https://github.com/andreasgrill/auto-selfcontrol
